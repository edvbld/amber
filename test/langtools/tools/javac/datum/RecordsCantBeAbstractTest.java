/*
 * @test /nodynamiccopyright/
 * @summary smoke negative test for datum classes
 * @compile/fail/ref=RecordsCantBeAbstractTest.out -XDrawDiagnostics RecordsCantBeAbstractTest.java
 */

abstract record RecordsCantBeAbstractTest(int i);
