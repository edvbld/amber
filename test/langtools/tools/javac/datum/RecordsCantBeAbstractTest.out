RecordsCantBeAbstractTest.java:7:1: compiler.err.record.cant.be.abstract
1 error
